jackal_description/urdf/accessories/vlp16_mount.urdf.xacro

```diff
@@ -91,7 +91,7 @@
       <origin xyz="0 0 ${height}" rpy="0 0 0" />
     </joint>
 
-    <xacro:VLP-16 parent="${prefix}_vlp16_plate" topic="${topic}">
+    <xacro:VLP-16 parent="${prefix}_vlp16_plate" name="velodyne" topic="/velodyne_points" organize_cloud="false" hz="10" samples="440" gpu="false">
```

Comment:
Revert to match our preference.

Reply:
I attempted to use just the original line 94 for this, and the LiDAR would not spawn with the `/velodyne_points` topic. After referring to the example urdf file from `velodyne_description`, I was able to see the `/velodyne_points` topic being published when applying the same xacro parameters. (Refer to [here](https://bitbucket.org/DataspeedInc/velodyne_simulator/src/1cfa7b3bb562849271b9185f3a636bae714aadda/velodyne_description/urdf/example.urdf.xacro#lines-38)).